### PR TITLE
Use prettier as code format on commit hooks

### PR DESCRIPTION
**Pull request by chregu**
Makes some nicer code, even though it has a different opinion on space-before-function-paren than eslint/standard

### CODE REVIEW

**Comment by pstadler** [comment]
Fyi, there’s https://github.com/prettier/prettier-eslint

**Comment by chregu** [comment]
Thanks for the hint. I changed to config to use that. Seems to work.

File: src/apis/memberships.js [hunk 1]
```diff
@@ -10,14 +9,14 @@
...
+export default state => {
   const memberships = {}
 
-  const ROLES = memberships.ROLES = {
+  const ROLES = (memberships.ROLES = {
     READ: 'read',
     WRITE: 'write',
     ADMIN: 'admin'
-  }
+  })
```
Comment (mweibel) [inline src/apis/memberships.js]: could you change that bit of code to following:

```javascript
const ROLES = {
  READ: 'read',
  WRITE: 'write',
  ADMIN: 'admin'
}
const memberships = { ROLES }
```

I think that's more easy on the eyes and doesn't have this weird new bracket which is added.

Comment (chregu) [reply src/apis/memberships.js]: fixed

**Pull request merged**